### PR TITLE
Add access log troubleshooting doc

### PR DESCRIPTION
**Pull request by lizan**

### CODE REVIEW

**Comment by geeknoid** [comment]
Deploy preview for *preliminary-istio* failed.

Built with commit f7f0a803a17ad776d85b06a7411ff79bb3e7c6d3

https://app.netlify.com/sites/preliminary-istio/deploys/5c58c5fc51c9a7000745c722

File: content/help/ops/traffic-management/troubleshooting/index.md [hunk 1]
```diff
@@ -6,6 +6,37 @@ weight: 30
 
 This section describes common problems and tools and techniques to address issues related to traffic management.
 
+## Requests are rejected by Envoy
+
+Requests may be rejected for various reasons, to identify the reason, you may inspect the access logs. By default,
```
Comment (geeknoid) [inline content/help/ops/traffic-management/troubleshooting/index.md]: reasons, and you may inspect the access logs to understand what is happening.

File: content/help/ops/traffic-management/troubleshooting/index.md [hunk 2]
```diff
@@ -6,6 +6,37 @@ weight: 30
...
+access logs are printed into standard output of the container, run the following command to see the log:
```
Comment (geeknoid) [inline content/help/ops/traffic-management/troubleshooting/index.md]: are printed -> are output
container, run -> container. Run

File: content/help/ops/traffic-management/troubleshooting/index.md [hunk 3]
```diff
@@ -6,6 +6,37 @@ weight: 30
...
+
+{{< text bash >}}
+$ kubectl logs -it PODNAME -c istio-proxy -n NAMESPACE
+{{< /text >}}
+
+In the default access log format, Envoy response flags and Mixer policy status are located after response code,
```
Comment (geeknoid) [inline content/help/ops/traffic-management/troubleshooting/index.md]: after the response code

File: content/help/ops/traffic-management/troubleshooting/index.md [hunk 4]
```diff
@@ -6,6 +6,37 @@ weight: 30
...
+if you are using custom log format, make sure to include `%RESPONSE_FLAGS%` and `%DYNAMIC_METADATA(istio.mixer:status)%`.
```
Comment (geeknoid) [inline content/help/ops/traffic-management/troubleshooting/index.md]: using a custom

File: content/help/ops/traffic-management/troubleshooting/index.md [hunk 5]
```diff
@@ -6,6 +6,37 @@ weight: 30
...
+
+Common response flags are:
+
+- `NR`: No route configured, check your `DestinationRule` or `VirtualService`.
+- `UO`: Upstream overflow with circuit breaking, check your circuit breaker configuration in `DestinationRule`.
+- `UF`: Failed to connect to upstream, if you're using Istio authentication, check your mutual TLS configuration too.
+
+The request is rejected by Mixer if you see `UAEX` response flag and Mixer policy status is not `-`.
+
+Common policy mixer statuses are:
```
Comment (geeknoid) [inline content/help/ops/traffic-management/troubleshooting/index.md]: mixer -> Mixer

File: content/help/ops/traffic-management/troubleshooting/index.md [hunk 6]
```diff
@@ -6,6 +6,37 @@ weight: 30
...
+if you are using custom log format, make sure to include `%RESPONSE_FLAGS%` and `%DYNAMIC_METADATA(istio.mixer:status)%`.
+
+Refer to the [Envoy response flags](https://www.envoyproxy.io/docs/envoy/latest/configuration/access_log#config-access-log-format-response-flags)
+for details of response flags.
+
+Common response flags are:
+
+- `NR`: No route configured, check your `DestinationRule` or `VirtualService`.
+- `UO`: Upstream overflow with circuit breaking, check your circuit breaker configuration in `DestinationRule`.
+- `UF`: Failed to connect to upstream, if you're using Istio authentication, check your mutual TLS configuration too.
```
Comment (frankbu) [inline content/help/ops/traffic-management/troubleshooting/index.md]: authentication, check for a [mutual TLS configuration conflict](#503-errors-after-setting-destination-rule).

File: content/help/ops/traffic-management/troubleshooting/index.md [hunk 7]
```diff
@@ -6,6 +6,37 @@ weight: 30
...
+
+The request is rejected by Mixer if you see `UAEX` response flag and Mixer policy status is not `-`.
```
Comment (frankbu) [inline content/help/ops/traffic-management/troubleshooting/index.md]: A request was rejected by Mixer if the response flag is `UAEX` and the Mixer policy status is not `-`.

**Comment by lizan** [comment]
@geeknoid @frankbu thanks for review, any further comments?

File: content/help/ops/traffic-management/troubleshooting/index.md [hunk 8]
```diff
@@ -6,6 +6,39 @@ weight: 30
 
 This section describes common problems and tools and techniques to address issues related to traffic management.
 
+## Requests are rejected by Envoy
+
+Requests may be rejected for various reasons, to identify the reasons, and you may inspect
```
Comment (geeknoid) [inline content/help/ops/traffic-management/troubleshooting/index.md]: Please update the start of the paragraph:

Requests may be rejected for various reasons. The best way to understand why requests are being rejected is by inspecting Envoy's access logs. By default, access logs...

Comment (lizan) [reply content/help/ops/traffic-management/troubleshooting/index.md]: Done

**Pull request merged**